lib/slack_coder/repo.ex

```diff
@@ -8,4 +8,10 @@ defmodule SlackCoder.Repo do
   def save(%Ecto.Changeset{model: %{id: _id}} = changeset) do
     update(changeset)
   end
+
+  # Fun addition
+  def count(queryable) do
+    import Ecto.Query
+    one(from q in queryable, select: count(q.id))
+  end
```

Comment:
@mtyeh411 Another cool bit here, doing a count is a bit awkward, i.e. instead of something simple like `Repo.count User`, you have to do `Repo.one (from u in User, select: count(u.id))`... But since `Repo` is just another module, we can add that here for us to make `Repo.count User` or even something like `Repo.count User.by_slack(["one", "two", "three"])` would work (even though itd be a dumb query to run).

Notice the locally scoped `import`, Elixir lets us do this! Now only this function has access to those additional methods and we don't pollute or possibly trample other things!